docs/api-publishing/08-ComplianceTesting.md

```diff
@@ -0,0 +1,179 @@
+    - **S**ecurity - For example,
+
+        >Given the user is authenticated
+        >
+        >And the user does not have permission to view sensitive information
+        >
+        >When the user attempts to access the sensitive information
+        > Then the sensitive information should not be displayed to the user
+    - **P**rivacy - For example,
+        >Given the [AsyncAPI Consumer](./04-AsyncAPI.md) receives an event of a patient death
+        >
+        >When the patient is not relevant to the Consumer system
+        >
+        >Then any information of the event should be discarded and not persisted or logged
+    - **I**dentity - For example,
+        >Given the API returns information
+        >
+        >And API consumer display the information in their user application
+        >
+        >And a user is not authenticated
+        >
+        >When the user tries to access the application
+        >
+        >Then the user is redirected to be authenticated with MFA before access is permitted
+    - **C**linical Safety - For example,
+        >Given the API returns the Patient immunisation information
+        >
+        >And regardless the viewing user's screen resolution 
+        >
+        >When the user views the data containing this immunisation information 
+        >
+        >Then the user is aware whether full information is viewed or there is undisplayed content
+    - **E**quity - For example, 
+        >Given the API returns data with macrons
+        >
+        >When the user views the data containing macrons
+        >
+        >Then the macron characters should render properly across all supported devices and platforms
+
+- Tests should reflect real life scenarios including edge cases (e.g. long names of patients) and data quality issues (e.g. non-compliant historical data or missing information.
+
+- Provide a mechanism to automate testing where possible
+
+- Tests can be extended with additional tests with incremental development 
+
+- Tests are reliable, repeatable, and traceable
+
+- Edge cases are considered (e.g. long names of patients, medicines, etc.) 
+
+- Remediation path is provided for test failure  
+
+- Recertification process is defined for periodic compliance testing and change of compliance status 
+
+- Provide examples and environment to demonstrate compliance (e.g. a compliance demo)
+
+- Provide test report or summary template  
+
+- Consider alternative or innovative tools to ease the compliance testing process, e.g. recorded demo, automation, report generation, etc. 
+
+## Compliance testing process 
+
+API Provider **MUST** provide compliant testing process definition. Diagrams **SHOULD** be used that describe an API Consumer compliance testing journey. 
+ 
+The process of compliance testing involves several steps to verify that the system meets the required standards. Below is an overview of a typical compliance testing process:
+```plantuml alt="Example of Compliance Testing Process"
```

Comment:
```suggestion
```plantuml alt="Flowchart illustrating the compliance testing process between an API Provider and API Consumer. It begins with the API Consumer identifying test requirements and guidelines, followed by a request for compliance testing. Test planning occurs with the API Provider preparing the test environment, while the API Consumer prepares the tests. The API Consumer then executes the tests and records the outcomes. Results are evaluated, and if necessary, remediation is performed. The cycle repeats until compliance is certified. Once certified, compliance monitoring is conducted by the API Provider and continues while ongoing."
```

Reply:
Should this be merged into the elaboration below the diagram?